Convert CRA demo and Readme to 0.44 (#6735)
microsoft/FluidFramework: examples/apps/cra-demo/README.md

There are two packages to install to get started with Fluid:

`@fluid-experimental/tinylicious-client` - Defines the service connection for our local Fluid server and starting schema for the [container](https://fluidframework.com/docs/glossary/#container)
`@fluid-experimental/frs-client` - Defines the client we'll use to connect to our Fluid [container](https://fluidframework.com/docs/glossary/#container), both locally and if deployed

> `@fluidframework/map` - Contains the SharedMap you will use to sync data

Reviewer: Shouldn't we use the uber package?

Author: fixed!

## 3. <a style="position: relative; top: 20px" name="import"></a> Import and Initialize Fluid Dependencies

> `FrsClient` is a client that will allow us to connect connect to a local `Tinylicious` server while testing our application. It also provides methods to create a [Fluid container](https://fluidframework.com/docs/glossary/#container) with a set of initial [DataObjects](https://fluidframework.com/docs/glossary/#dataobject) or [DDSes](https://fluidframework.com/docs/concepts/dds/) that are defined in the `containerSchema`.

Reviewer: ```suggestion
`FrsClient` is a client that will allow us to connect connect to a local `Tinylicious` server while testing our application. It also provides methods to create a [Fluid container](https://fluidframework.com/docs/glossary/#container) with a set of initial [DDSes](https://fluidframework.com/docs/concepts/dds/) or [DataObjects](https://fluidframework.com/docs/glossary/#dataobject) that are defined in the `containerSchema`.
```

Leading with ddses since that's what they're using in this example.

};
```

### 3.b Configure the service client

> In this demo we will only be dealing with a local configuration, but in a production environment you could swap this out with a production configuration without changing your application.

Reviewer: ```suggestion
This demo illustrates using a local configuration, but in a production environment you could swap this out with a production configuration without changing your application.
```

```js
// add below getContainerId
const localConfig = {
    tenantId: "local",

>     tokenProvider: new InsecureTokenProvider("tenantId", { id: "userId" }),

Reviewer: ```suggestion
    tokenProvider: new InsecureTokenProvider("anyValue", { id: "userId" }),
```
Suggest changing the value to make it clear that the value is irrelevant.

const localConfig = {
    tenantId: "local",
    tokenProvider: new InsecureTokenProvider("tenantId", { id: "userId" }),
    orderer: "http://localhost:7070",
    storage: "http://localhost:7070",
};
```

> ## 4. <a style="position: relative; top: 20px" name="init"></a> Get Fluid Data

Reviewer: Is the style needed?

Author: it makes the jump links work properly

Author: ![image](https://user-images.githubusercontent.com/1434956/125840772-610fb966-e0fc-43b3-aed7-ae9ec8b73b94.png)

> Before we can access any Fluid data, we need to create our configured FRS `client` and define our container schema.

Reviewer: ```suggestion
Before you can access any Fluid data, you need to create a  configured FRS `client` and define your container schema.
```

- `containerSchema` is going to include a string `name` and a collection of the data types our application will use.

The following `getFluidData` function utilizes the `getContainerId` to return a unique ID and determine if this is an existing document (`getContainer`) or if we need to create a new one (`createContainer`).

> Since this `getFluidData` function is an async, we'll need to wait for the `initialObjects` to be returned. Once returned, each `initialObjects` key will point to a connected data structure as defined in the schema.

Reviewer: ```suggestion
Since `getFluidData` is an async function, we'll need to `await` for the `initialObjects` to be returned. Once returned, each `initialObjects` key will point to a connected data structure as defined in the schema.
```
